tests/python/gaia-ui-tests/gaiatest/apps/ftu/app.py

```diff
@@ -117,6 +117,9 @@ def tap_next(self):
             expected.element_present(*self._next_button_locator))
         Wait(self.marionette).until(expected.element_displayed(next_button))
         Wait(self.marionette).until(expected.element_enabled(next_button))
+        # In b2g desktop builds, this sleep prevents intermittent failures
+        import time
```

Comment:
Please move the import to the top of the file. I know style varies in a few of our files, but standard is generally to import globally.